sample/bin/install.js

```diff
@@ -0,0 +1,20 @@
+const { exec } = require('@effection/node');
+const { subscribe } = require('@effection/subscription');
+const { spawn } = require('effection');
+const { yarn } = require('./constants');
+
+function* install({ cwd, stdio }) {  
+  let command = yarn ? 'yarn' : 'npm';
+  let install = yield exec(`${command} install`, { cwd });
+  if(stdio === 'inherit'){
+    yield spawn(subscribe(install.stdout).forEach((data) => {
+      process.stdout.write(data);
+      return Promise.resolve();
+    }));
+  };
```

Comment:
We should also pipe `stderr` if we're inheriting.
```suggestion
  if(stdio === 'inherit') {
    yield spawn(subscribe(install.stdout).forEach((data) => {
      process.stdout.write(data);
      return Promise.resolve();
    }));
    yield spawn(subscribe(install.stderr).forEach((data) => {
      process.stderr.write(data);
      return Promise.resolve();
    }));
  };
```

Reply:
done 👍